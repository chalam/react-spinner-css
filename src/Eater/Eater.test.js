import React from 'react'
import renderer from 'react-test-renderer'
import { Eater } from '..'

it('renders correctly', () => {
  const tree = renderer.create(<Eater />).toJSON()
  expect(tree).toMatchSnapshot()
})
