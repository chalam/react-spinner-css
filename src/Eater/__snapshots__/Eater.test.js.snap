// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Array [
  <style>
    $
    [object Object]
  </style>,
  <div
    class="loadingio-spinner-bean-eater-deh6e0vw0xt"
  >
    <div
      class="ldio-mtx3eu3brlo"
    >
      <div>
        <div />
        <div />
        <div />
      </div>
      <div>
        <div />
        <div />
        <div />
      </div>
    </div>
  </div>,
]
`;
